docs/bastion.md

```diff
@@ -2,61 +2,84 @@
 
 Bastion provide an external facing point of entry into a network containing private network instances. This host can provide a single point of fortification or audit and can be started and stopped to enable or disable inbound SSH communication from the Internet, some call bastion as the "jump server".
 
-Note: Bastion will get setup for the cluster(by default) only when `--topology="private"`.
-
 * [More information on bastion from aws](http://docs.aws.amazon.com/quickstart/latest/linux-bastion/architecture.html)
 * [More information on bastion from gce](https://cloud.google.com/solutions/connecting-securely#bastion)
 
 ## AWS
 
-### Specify instance type of bastion
+### Enable/Disable bastion
 
-Instance types in AWS comprise varying combinations of CPU, memory, storage, and networking capacity and give you the flexibility to choose the appropriate mix of resources for your applications.
+To enable a bastion instance group, a user will need to set the `--bastion` flag on cluster create
 
-- **Defaults** to `t2.medium`
-- **Configure:** Bastion Instance type can be modified using `kops edit cluster`
 ```
-topology:
-    bastion:
-        MachineType: c4.large
+kops create cluster --topology private --networking $provider --bastion $NAME
 ```
-[More information](https://aws.amazon.com/ec2/instance-types/)
 
+### Configure the bastion instance group
 
-### Turn on/off bastion
+You can edit the bastion instance group to make changes. By default the name of the bastion instance group will be `bastions` and you can specify the name of the cluster with `--name` as in:
 
-To turn on/off bastion host setup completely.
-- **Defaults** to `false` if the topology selected is `public`
-- **Defaults** to `true` if the topology selected is `private`
-- **Configure:**
 ```
-kops create cluster --bastion=[true|false]
+kops edit ig bastions --name $KOPS_NAME
 ```
-OR using `kops edit cluster`
+
+You should now be able to edit and configure your bastion instance group.
+
 ```
-topology:
-    bastion:
-        Enable: true
+apiVersion: kops/v1alpha2
+kind: InstanceGroup
+metadata:
+  creationTimestamp: "2017-01-05T13:37:07Z"
+  name: bastions
+spec:
+  associatePublicIp: true
+  image: kope.io/k8s-1.4-debian-jessie-amd64-hvm-ebs-2016-10-21
+  machineType: t2.micro
+  maxSize: 1
+  minSize: 1
+  role: Bastion
+  subnets:
+  - utility-us-east-2a
 ```
 
-### Reach bastion from outside of vpc using a name
+**Note**: If you want to turn off the bastion server, you must set the instance group `maxSize` and `minSize` fields to `0`.
+
+If you do not want the bastion instance group created at all, simply drop the `--bastion` flag off of your create command. The instance group will never be created.
```

Comment:
We are defaulting to not creating a bastion instance group.. defaulting here is weird.. and without hacking the args myself it would be hard to tell what the user was actually saying or not.. this seemed like the safest and simplest choice

How does one regenerate the cobra docs?